--- virt-manager-0.9.1/src/virtManager/manager.py~	2012-02-01 00:14:18.000000000 +0100
+++ virt-manager-0.9.1/src/virtManager/manager.py	2012-02-20 00:17:05.028801540 +0100
@@ -661,11 +661,6 @@
             details = "%s\n\n%s" % (msg, tb)
 
         else:
-            hints = []
-            if conn.is_remote() and re.search(r"nc: .* -- 'U'", details):
-                hints.append(
-                    _("\n - The remote netcat understands the '-U' option"))
-
             if warnconsole:
                 msg += _("Could not detect a local session: if you are \n"
                          "running virt-manager over ssh -X or VNC, you \n"
@@ -674,8 +669,6 @@
             else:
                 msg += _("Verify that:\n" +
                          " - The 'libvirtd' daemon has been started")
-                for hint in hints:
-                    msg += hint
 
             msg = msg.strip("\n")
             details = (("%s\n\n" % msg) +
